pkg/daemon/admission/server.go

```diff
@@ -17,19 +17,18 @@ limitations under the License.
 package admission
 
 import (
-	"crypto/tls"
-	"fmt"
 	"github.com/coreos/pkg/capnslog"
 	"github.com/rook/rook/pkg/clusterd"
-	"net/http"
-	"path/filepath"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
 )
 
 const (
 	tlsDir            = "/etc/webhook"
 	tlsCertFile       = "tls.crt"
 	tlsKeyFile        = "tls.key"
 	serverPort  int32 = 8079
+	port              = 8079
```

Comment:
What is the difference between `serverPort` and `port`? Can't we just re-use `serverPort`?

Reply:
Removed ! Moved the dir and port to cmd/ceph/admission.go